ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/sdn/workers-rhel8-fips/provision/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-provision-chain.yaml

```diff
@@ -1,9 +1,10 @@
 chain:
-  as: cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-provision
+  as: cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-provision
```

Comment:
To be more accurate, if this `fips` present in chain name,  we should add `FIPS_ENABLED: "true"` to the chain (although it's already set in job config)

Reply:
Ah, the same comment as https://github.com/openshift/release/pull/44811/files#r1371436518, and I think we should remove `FIPS_ENABLED` from job config.